### ISSUE TITLE
Rpm build warnings about files listed twice

### ISSUE THREAD

**Issue by DrDaveD** [apptainer/singularity#2482]
## Version of Singularity:

Current master branch, after PR #2401.

### Expected behavior

Clean rpm builds.

### Actual behavior

There are these rpm build warnings:
```
warning: File listed twice: /etc/singularity/actions/exec
warning: File listed twice: /etc/singularity/actions/run
warning: File listed twice: /etc/singularity/actions/shell
warning: File listed twice: /etc/singularity/actions/start
warning: File listed twice: /etc/singularity/actions/test
```

### Steps to reproduce behavior

make -C builddir rpm

**Comment by PeterWhittaker** [comment]
Please refer to the commit above: A %dir macro specific to ...singularity/actions/... may eliminate the warnings.

**Comment by dtrudg** [comment]
This issue will be removed by https://github.com/sylabs/singularity/pull/4938/files which removes the action scripts into code.

**Comment by dtrudg** [comment]
Closed via #4938 removal of action scripts
